code/obj/item/material.dm

```diff
@@ -494,14 +495,16 @@
 	name = "scrap"
 	desc = "Some twisted and ruined metal. It could probably be smelted down into something more useful."
 	icon_state = "scrap"
+	stack_type = /obj/item/raw_material/scrap_metal
 	burn_possible = 0
+	set_name = TRUE
+	material_name = "Steel"
 
 	New()
 		..()
 		icon_state += "[rand(1,5)]"
-
-/obj/item/raw_material/scrap_metal/steel
-	material_name = "Steel"
+	steel
+		material_name = "Steel"
```

Comment:
:/

Reply:
i can change that back, not sure why i did it, but can you explain whats wrong with it?

Reply:
Changing an absolute definition to a relative one

Reply:
ok! I'm not knowledgable enough to know the difference or what it changes in this scenario